### DISCUSSION TITLE
希望v2把v1的简洁版首页添加回来

### DISCUSSION THREAD

**Discussion by EasyRain** [the1812/Bilibili-Evolved#2523] [功能建议 / Ideas]
v2就只有一个简化首页，感觉简化了也不好看，还是喜欢v1的简洁版首页，圆润些，而且一目了然

**Comment by Geeyun-JY3** [comment]
等，有计划的：#2448
> 暂不支持 v1 的 `简化首页` 功能, 在 v2 中 `简化首页` 等于 v1 的 `首页过滤` 功能. v1 的 `简化首页` 由于代码设计过于糟糕, 很难再进一步扩充功能, 我计划在 v2 发布后重写这个功能, 届时会比 v1 更加美观和强大. 如果你非常需要这个功能, 请暂时不要升级到 v2.
